ruby/README.md

```diff
@@ -1,55 +1,61 @@
+
+**Disclaimer:** Given the ever updating nature of the curriculum, the outline might be outdated. See the [Ruby course on the website](https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby)
+instead.
+
+### Introduction
+  1. [How this Course Will Work](introduction/how_this_course_will_work.md)
+  2. [Installing Ruby](introduction/installing_ruby.md)
+### Basic Ruby
+  1. [Basic Data Types](basic_ruby/basic_data_types.md)
+  2. [Variables](basic_ruby/variables.md)
+  3. [Input and Output](basic_ruby/input_and_output.md)
+  4. [Conditional Logic](basic_ruby/conditional_logic.md)
+  5. [Loops](basic_ruby/loops.md)
+  6. [Arrays](basic_ruby/arrays.md)
+  7. [Hashes](basic_ruby/hashes.md)
+  8. [Methods](basic_ruby/methods.md)
+  9. [Debugging](basic_ruby/debugging.md)
+  10. [Basic Enumerable Methods](basic_ruby/basic_enumerable_methods.md)
+  11. [Predicate Enumerable Methods](basic_ruby/predicate_enumerable_methods.md)
+  12. [Nested Collections](basic_ruby/nested_collections.md)
+### Basic Ruby Projects
+  1. [**Project: Caesar Cipher**](basic_ruby_projects/project_caesar_cipher.md)
+  2. [**Project: Sub Strings**](basic_ruby_projects/project_sub_strings.md)
+  3. [**Project: Stock Picker**](basic_ruby_projects/project_stock_picker.md)
+  4. [**Project: Bubble Sort**](basic_ruby_projects/project_bubble_sort.md)
+### Object Oriented Programming Basics
+  1. [Object Oriented Programming](object_oriented_programming_basics/object_oriented_programming.md)
+  2. [**Project: Tic Tac Toe**](object_oriented_programming_basics/project_tic_tac_toe.md)
+  3. [**Project: Mastermind**](object_oriented_programming_basics/project_mastermind.md)
+### Files and Serialization
+  1. [Files and Serialization](files_and_serialization/files_and_serialization.md)
+  2. [**Project: Event Manager**](files_and_serialization/project_event_manager.md)
+  3. [**Project: Hangman**](files_and_serialization/project_hangman.md)
+### Advanced Ruby
+  1. [Blocks](advanced_ruby/blocks.md)
+  2. [Pattern Matching](advanced_ruby/pattern_matching.md)
+  3. [**Project: Custom Enumerables**](advanced_ruby/project_custom_enumerables.md)
+### A Bit of Computer Science
+  1. [A Very Brief Intro to CS](computer_science/a_very_brief_intro_to_cs.md)
+  2. [Recursive Methods](computer_science/recursive_methods.md)
+  3. [**Project: Recursion**](computer_science/project_recursion.md)
+  4. [Common Data Structures and Algorithms](computer_science/common_data_structures_algorithms.md)
+  5. [**Project: Linked Lists**](computer_science/project_linked_lists.md)
+  6. [**Project: Binary Search Trees**](computer_science/project_binary_search_trees.md)
+  7. [**Project: Knight Travails**](computer_science/project_knights_travails.md)
+### Intermediate Git
+  1. [A Deeper Look at Git](../git/intermediate_git/a_deeper_look_at_git.md)
+  2. [Using Git in the Real World](../git/intermediate_git/using_git_in_the_real_world.md)
```

Comment:
I should've mentioned this in the issue. We don't need to include the links for the Intermediate Git section here. It's already covered in the [git's folder README](https://github.com/TheOdinProject/curriculum/blob/main/git/README.md).

Can you please remove this section from here?

Reply:
Done! I removed it in my most recent commit